fe/src/main/java/org/apache/doris/qe/ConnectProcessor.java

```diff
@@ -126,6 +129,11 @@ private void auditAfterExec(String origStmt, StatementBase parsedStmt, PQuerySta
                 MetricRepo.HISTO_QUERY_LATENCY.update(elapseMs);
             }
             ctx.getAuditEventBuilder().setIsQuery(true);
+            ctx.getQueryDetail().setEventTime(endTime);
```

Comment:
Can you explain why set eventTime as endTime?

Reply:
Because one query will be set when started, and then update when finished.
If query monitor collect between the two point. The query monitor can use eventTime to distinguish the two event.